sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/SdkMeterProvider.java

```diff
@@ -98,10 +98,11 @@ public static SdkMeterProviderBuilder builder() {
    * AggregationFactory aggregationFactory = AggregationFactory.minMaxSumCount();
    *
    * //register the view with the MeterSdkProvider
-   * meterProvider.registerView(instrumentSelector, aggregationFactory);
+   * meterProvider.registerView(instrumentSelector, View.builder()
+   *   .setAggregatorFactory(factory).build());
```

Comment:
s/factory/aggregationFactory? To make example work.

Reply:
done